editor/osm_feature_matcher.cpp

```diff
@@ -0,0 +1,145 @@
+#include "base/logging.hpp"
+
+#include "editor/osm_feature_matcher.hpp"
+
+namespace osm
+{
+double constexpr kPointDiffEps = MercatorBounds::GetCellID2PointAbsEpsilon();
+
+bool LatLonEqual(ms::LatLon const & a, ms::LatLon const & b)
+{
+  return a.EqualDxDy(b, kPointDiffEps);
+}
+
+double ScoreLatLon(XMLFeature const & xmlFt, ms::LatLon const & latLon)
+{
+  auto const a = MercatorBounds::FromLatLon(xmlFt.GetCenter());
+  auto const b = MercatorBounds::FromLatLon(latLon);
+  return 1.0 - (a.Length(b) / kPointDiffEps);
```

Comment:
What if a.Length(b) / kPointDiffEps is greater than 1.0? Why not to clamp return value?